### PR TITLE
drop sql test

### PR DESCRIPTION
**Pull request by AngLi-Leon**

### CODE REVIEW

**Comment by apavlo** [comment]
Why did you close this?

**Comment by AngLi-Leon** [comment]
I find a mistake in my test case. Will change it and PR again.

**Comment by apavlo** [comment]
No, just keep the same PR and push a new commit. No need to close/open a new one.

File: test/sql/drop_sql_test.cpp [hunk 1]
```diff
@@ -0,0 +1,76 @@
...
+#include "planner/create_plan.h"
+
+#include "sql/sql_tests_util.h"
+
+namespace peloton {
+namespace test {
+
+class DropSQLTests : public PelotonTest {};
+
+TEST_F(DropSQLTests, DropSQLTest) {
```
Comment (apavlo) [inline test/sql/drop_sql_test.cpp]: Rename to DropTableTest

File: test/sql/drop_sql_test.cpp [hunk 2]
```diff
@@ -0,0 +1,76 @@
...
+
+  // Insert and query from that table
+  SQLTestsUtil::ExecuteSQLQuery("INSERT INTO test VALUES (1, 10);", result,
+                                tuple_descriptor, rows_affected, error_message);
+  SQLTestsUtil::ExecuteSQLQuery("SELECT * FROM test;", result, tuple_descriptor,
+                                rows_affected, error_message);
+  EXPECT_EQ(result[0].second[0], '1');
+
+  // Drop the table
+  SQLTestsUtil::ExecuteSQLQuery("DROP TABLE test;");
```
Comment (apavlo) [inline test/sql/drop_sql_test.cpp]: Does this query return a result? If so, then we should check it.

File: test/sql/drop_sql_test.cpp [hunk 3]
```diff
@@ -0,0 +1,76 @@
...
+TEST_F(DropSQLTests, DropSQLTest) {
+  catalog::Catalog::GetInstance()->CreateDatabase(DEFAULT_DB_NAME, nullptr);
+
+  // Create a table first
+  SQLTestsUtil::ExecuteSQLQuery("CREATE TABLE test(a INT PRIMARY KEY, b INT);");
+
+  // Check the table in catalog
+  EXPECT_EQ(catalog::Catalog::GetInstance()
+                ->GetDatabaseWithName(DEFAULT_DB_NAME)
+                ->GetTableCount(),
```
Comment (apavlo) [inline test/sql/drop_sql_test.cpp]: You should look for the actual Table instead of just getting the count in case we start adding internal tables.

**Pull request merged**